packages/paste-core/components/summary-detail/stories/index.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import type { StoryFn } from "@storybook/react";
+import { Box } from "@twilio-paste/box";
+import { Paragraph } from "@twilio-paste/paragraph";
+import * as React from "react";
+
+import {
+  SummaryDetail,
+  SummaryDetailContent,
+  SummaryDetailHeading,
+  type SummaryDetailInitialState,
+  type SummaryDetailStateReturn,
+  useSummaryDetailState,
+} from "../src";
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: "Components/SummaryDetail",
```

Comment:
suggestion: I have this issue where I don't click save in Jira and my update doesn't persist. But I added an example from Flex so that we could create a real use case example story for VRT'ing.

Reply:
Thanks for that, changed up the design and examples